site/content/docs/commands/task-run.en.md

```diff
@@ -20,44 +20,59 @@ Generally, the steps involved in task run are:
--n, --task-group-name string         Optional. The group name of the task. Tasks with the same group name share the same set of resources.
-    --task-role string               Optional. The role for the task to use.
-```
-## Example
+Name Flags
+  -n, --task-group-name string   Optional. The group name of the task. 
+                                 Tasks with the same group name share the same set of resources. 
+                                 (default directory name)
+
+Build Flags
+      --build-context string   Path to the Docker build context.
+                               Mutually exclusive with -i, --image.
+      --dockerfile string      Path to the Dockerfile.
+                               Mutually exclusive with -i, --image. (default "Dockerfile")
+  -i, --image string           The location of an existing Docker image.
+                               Mutually exclusive with -d, --dockerfile.
+      --tag string             Optional. The container image tag in addition to "latest".
+
+Placement Flags
+      --app string                Optional. Name of the application.
+                                  Cannot be specified with 'default', 'subnets' or 'security-groups'.
+      --cluster string            Optional. The short name or full ARN of the cluster to run the task in. 
+                                  Cannot be specified with 'app', 'env' or 'default'.
+      --default                   Optional. Run tasks in default cluster and default subnets. 
+                                  Cannot be specified with 'app', 'env' or 'subnets'.
+      --env string                Optional. Name of the environment.
+                                  Cannot be specified with 'default', 'subnets' or 'security-groups'.
+      --security-groups strings   Optional. Additional security group IDs for the task to use. Can be specified multiple times.
+      --subnets strings           Optional. The subnet IDs for the task to use. Can be specified multiple times.
+                                  Cannot be specified with 'app', 'env' or 'default'.
+
+Task Configuration Flags
+      --acknowledge-secrets-access     Optional. Skip the confirmation question and grant access to the secrets specified by --secrets flag. 
+                                       This flag is useful only when 'secrets' flag is specified
+      --command string                 Optional. The command that is passed to "docker run" to override the default command.
+      --count int                      Optional. The number of tasks to set up. (default 1)
+      --cpu int                        Optional. The number of CPU units to reserve for each task. (default 256)
+      --entrypoint string              Optional. The entrypoint that is passed to "docker run" to override the default entrypoint.
+      --env-vars stringToString        Optional. Environment variables specified by key=value separated by commas. (default [])
```

Comment:
```suggestion
      --env-vars key=value        Optional. Environment variables specified by key=value separated by commas. (default [])
```

Reply:
That's unfortunately what cobra generates 😢 so I kept it as is and we have the long description mention "key=value"